server/server.ts

```diff
@@ -23,12 +29,17 @@ class Server {
     this.app.use(express.json());
     this.app.use(express.urlencoded({ extended: true }));
     this.app.use(cookieParser());
+    this.app.use(attachUser);
```

Comment:
We want to make sure this happens _after_ we parse cookies, but _before_ we handle any of the API routes, since those might be dependent on `req.user`.